docs/summits/2025_03/call_to_action.md

```diff
@@ -1 +1,32 @@
 # We need your help !
+
+Thank you for attending this WheelNext Summit (or reading its website materials)!
+Your participation in and contributions during and after this event matter -
+we're collectively trying to achieve some ambitious goals, and we'll need all
+the help we can get.
+
+A first goal is community building - WheelNext is young, and we're rapidly trying
+to pull together as many stakeholders as possible: companies, open source projects,
+and individual Python packaging contributors. Meeting others with similar goals
+and pain points helps collaborate in the future, and building an understanding
+of pain points different from our own will lead to better discussions and
+solutions.
+
+Concretely, here are ways to contribute. During the event:
+
+- Share your own goals, pain points, and ideas
+- Pick one of the 3 workshops to attend and collaborate on a small concrete deliverable
```

Comment:
We decided to move to 2 workshops only - Barry withdrew his to not distract from the two major topics.